crates/matrix-sdk/src/client/mod.rs

```diff
@@ -66,6 +68,8 @@ use ruma::{
     ServerName, UInt,
 };
 use serde::de::DeserializeOwned;
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Note: this is one of the cases, where I'd rather have the two imports next to one another to make that cfg-relationship obvious than insisting on a alphabetical order...

Reply:
rustfmt resorted them.